build.sbt

```diff
@@ -41,11 +41,11 @@ mimaPreviousArtifacts := Set.empty
 crossScalaVersions := List()
 addCommandAlias(
   "scalafixAll",
-  "; ++2.12.10 ; scalafixEnable ; all scalafix test:scalafix"
```

Comment:
before https://github.com/scalacenter/sbt-scalafix/pull/292, the patch version was ignored as sbt-scalafix uses to set its own version ([2.13 or later, containing `nowarn`](https://github.com/scala/scala/commit/ec8f9de10ca9be1a3b9ec4b0c039a24455f7bad5))